Hymns/Display/HymnLyricsView.swift

```diff
@@ -36,7 +36,7 @@ public struct HymnLyricsView: View {
                         ForEach(lyrics, id: \.self) { verseViewModel in
                             VerseView(viewModel: verseViewModel, transliterate: self.$transliterate)
                         }
-                    }.padding()
+                    }.frame(minWidth: 0, maxWidth: .infinity, alignment: .leading).padding()
```

Comment:
I think this is the key line. Everything else, I removed it, and it seemed to still work

Reply:
Yeah, that is the line.